debug-scripts/packages

```diff
@@ -2,5 +2,6 @@
 set -ux
 
 dpkg --list > $DEBUG_SCRIPT_DIR/dpkg-list
+snap list > $DEBUG_SCRIPT_DIR/snap-list
```

Comment:
Do we know if snap is supported on trusty and if this will work on non-xenial+ hosts?